test/integration/patients/patient/patient-flow.js

```diff
@@ -209,6 +210,7 @@ context('patient flow page', function() {
         expect($action.find('[data-owner-region]')).to.contain('NUR');
         expect($action.find('[data-due-day-region] .is-overdue')).to.exist;
         expect($action.find('[data-form-region]')).not.to.be.empty;
+        expect($action.find('.patient__action-attachment')).to.exist;
```

Comment:
I think we should test for the icon directly over the style class

Reply:
Updated in last push.